Commands/escaperooms.py

```diff
@@ -0,0 +1,548 @@
+import discord
+import pandas as pd
+import re
+import time
+import asyncio
+
+from datetime import datetime
+from discord.ext import commands
+from .GuildData import get_guild_data
+from Helpers.constants import *
+from Helpers.common_functions import usernames_from_ids
+from Translations.Translations import get_text as translate
+from Helpers.TranslationHelper import get_culture_from_context as culture
+from Helpers.emoji import white_check_mark, scales, trophy, thumbs_down, thumbs_up
+from Helpers.log import *
+
+
+class Escaperooms(commands.Cog, name="Escaperooms"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    def read_tracking_table(self) -> pd.DataFrame:
```

Comment:
Function needs an extra piece that checks if a file exists, if not it should create a Pandas dataframe with the correct basic columns (Naam,Discord ID,Heeft tabletop,dummyroom) and call `write_tracking_table`